File: streampipes-model/src/main/java/org/streampipes/model/modelconnect/DomainPropertyProbability.java
package org.streampipes.model.modelconnect;

import org.streampipes.empire.annotations.Namespaces;
import org.streampipes.empire.annotations.RdfProperty;
import org.streampipes.empire.annotations.RdfsClass;
import org.streampipes.model.base.NamedStreamPipesEntity;
import org.streampipes.model.base.UnnamedStreamPipesEntity;

import javax.persistence.Entity;

@Namespaces({"sp", "https://streampipes.org/vocabulary/v1/"})
@RdfsClass("sp:DomainPropertyProbability")
@Entity
public class DomainPropertyProbability extends UnnamedStreamPipesEntity {

    @RdfProperty("sp:domainProperty")
    private String domainProperty;


    @RdfProperty("sp:probability")
    private String probability;

    public DomainPropertyProbability() {
        super();
    }

    public DomainPropertyProbability(String domainProperty,  String probability) {
        this.domainProperty = domainProperty;
        this.probability = probability;
    }

    public String getDomainProperty() {
        return domainProperty;
    }

    public void setDomainProperty(String domainProperty) {
        this.domainProperty = domainProperty;
    }

    public String getProbability() {
        return probability;
    }

    public void setProbability(String probability) {
        this.probability = probability;
    }
}

File: src/app/connect/schema-editor/model/DomainPropertyProbability.ts
import {RdfsClass} from '../../../platform-services/tsonld/RdfsClass';
import {RdfProperty} from '../../../platform-services/tsonld/RdfsProperty';
import {RdfId} from '../../../platform-services/tsonld/RdfId';

@RdfsClass('sp:DomainPropertyProbability')
export class DomainPropertyProbability {

    private static serialVersionUID = -3994041794693686406;

    @RdfId
    public id: string;

    @RdfProperty('sp:domainProperty')
    public domainProperty: String;


    @RdfProperty('sp:probability')
    public probability: String;

    constructor () {
    }
}